tests/cases/fourslash/unusedImports14FS.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path='fourslash.ts' />
+
+// @noUnusedLocals: true
+// @Filename: file2.ts
+//// [| import /* 1 */ A /* 2 */, /* 3 */ { x } from './a'; |]

```

Comment:
Ha, I guess I got tired of typing out comments. I didn't think it was interesting to test since we're removing the ```{ x }``` node altogether.  But might as well.